meta-lmp-bsp/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bbappend

```diff
@@ -2,3 +2,17 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 PROVIDES:uz3eg-iocc-sec = "virtual/boot-bin"
 SRC_URI:uz3eg-iocc-sec = "file://boot.bin"
+
+DEPENDS:append:tegra = "tegra-uefi-capsules"
+RDEPENDS:${PN}:append:tegra = "setup-nv-boot-control"
+SRC_URI:append:tegra = " \
```

Comment:
I plan to make it generic, include this scripts when`efi` MACHINE_FEATURE is enabled